incident.go

```diff
@@ -544,17 +544,20 @@ func (c *Client) CreateIncidentNote(id string, note IncidentNote) error {
 // period of time.
 //
 // Deprecated: Use SnoozeIncidentWithContext instead.
-func (c *Client) SnoozeIncidentWithResponse(id string, duration uint) (*Incident, error) {
-	return c.SnoozeIncidentWithContext(context.Background(), id, duration)
+func (c *Client) SnoozeIncidentWithResponse(id, from string, duration uint) (*Incident, error) {
+	return c.SnoozeIncidentWithContext(context.Background(), id, from, duration)
 }
 
 // SnoozeIncidentWithContext sets an incident to not alert for a specified period of time.
-func (c *Client) SnoozeIncidentWithContext(ctx context.Context, id string, duration uint) (*Incident, error) {
-	d := map[string]uint{
+func (c *Client) SnoozeIncidentWithContext(ctx context.Context, id, from string, duration uint) (*Incident, error) {
+	headers := map[string]string{
+		"from": from,
```

Comment:
nit: use upper-case header names to be consistent with the other methods and the documentation.
```suggestion
		"From": from,
```

Reply:
Fixed